src/main/java/com/faforever/api/data/domain/Login.java

```diff
@@ -63,6 +65,14 @@
   @ReadPermission(expression = IsEntityOwner.EXPRESSION + " OR " + ReadAccountPrivateDetailsCheck.EXPRESSION)
   private String steamId;
 
+  @OneToMany
+  @JoinTable(name = "unique_id_users",
+    joinColumns = @JoinColumn(name = "user_id"),
+    inverseJoinColumns = @JoinColumn(name = "uniqueid_hash", referencedColumnName = "hash")
+  )
+  @ReadPermission(expression = IsEntityOwner.EXPRESSION + " OR " + ReadAccountPrivateDetailsCheck.EXPRESSION)
```

Comment:
Users should not be able to see their uniqueids. I know eventually it can't be prevented as you can snoop traffic or run it locally, but we have to make more difficult than a rest call.
So no IsEntityOwner here please.

Reply:
Aight. Done.